config.json

```
@@ -1393,6 +1393,17 @@
         "strings"
       ]
     },
+    {
+        "slug": "dnd-character",
+        "uuid": "58625685-b5cf-4e8a-b3aa-bff54da0689d",
+        "core": false,
+        "unlocked_by": null,
```

Since this is the second exercise in this track that deals with `randomness`, I would slot it after `robot-name`:

```suggestion
        "unlocked_by": "robot-name",
```